slides/intro-co-cekat.md

```diff
@@ -19,6 +22,10 @@
 ## Analytička
 ## UX designérka
 
+>>>
+
+Nejdříve je potřeba prozkoumat pro koho se to dělá, poznat zákazníky, budoucí čtenáře nebo uživatele. Už se podaří cílovou skupinu webu popsat, tak designéři mohou navrhovat.
```

Comment:
…uživatele. **Jakmile** se podaří…

Reply:
Opraveno.

Reply:
@Kout Oprava je v mém forku. Mám udělat další PR nebo to umíš mergnout?